.github/workflows/main.yaml

```diff
@@ -0,0 +1,145 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - add-ci
```

Comment:
It seems this is for testing?

Reply:
Yes, but the last commit changes Ci to `main` branch.